owncloudData/src/main/java/com/owncloud/android/data/webfinger/datasources/implementation/OCWebfingerRemoteDatasource.kt

```diff
@@ -20,19 +20,51 @@ package com.owncloud.android.data.webfinger.datasources.implementation
 import com.owncloud.android.data.ClientManager
 import com.owncloud.android.data.executeRemoteOperation
 import com.owncloud.android.data.webfinger.datasources.WebfingerRemoteDatasource
+import com.owncloud.android.domain.webfinger.model.WebfingerRel
+import com.owncloud.android.lib.common.OwnCloudClient
+import com.owncloud.android.lib.common.authentication.OwnCloudCredentialsFactory
 import com.owncloud.android.lib.resources.webfinger.services.WebfingerService
 
 class OCWebfingerRemoteDatasource(
```

Comment:
If you finally change the name of the interface, rename this consequently as well 😁